tools/tests/form-control/test.js

```diff
@@ -12,15 +14,19 @@ const formsIds = {
 	SWITCH_FORM: 'switch-form',
 	DROPDOWN_FORM: 'dropdown-form',
 }
-const RESPONSE_CONTAINER_ID = 'form-response';
-const forms = [
-	{
-		testName: 'Login form',
-		id: formsIds.LOGIN_FORM,
-		shouldAddRequestFinishCallback: true,
-		submitData: '{"username":"name","password":"pass","poll":"cats"}',
-		template:
-			`<div class="form-wrapper">
+
+const formsTestNames = {
+	LOGIN_FORM: 'Login form',
+	REGISTER_FORM: 'Register form',
+	PREVENT_SUBMIT_FORM: 'Prevent form from submiting',
+	CHECKBOXES_FORM: 'Checkbox cases form',
+	RADIO_FORM: 'Radio cases form',
+	SWITCH_FORM: 'Switch cases form',
+	DROPDOWN_FORM: 'Dropdown cases form',
+}
+
+const formsTemplates = {
```

Comment:
We are going to take a task to move various resources for the tests in another file. Since it is not clear how we are going to resolve the problem -> having multiple files or a general one, I will hold back on resolving this here.